lib/screens/v2/candidate_generator/dup/departures.ex

```diff
@@ -458,8 +460,18 @@ defmodule Screens.V2.CandidateGenerator.Dup.Departures do
       MapSet.disjoint?(not_informed, informed_stop_ids)
   end
 
-  # If we are currently overnight, returns the first schedule of the day for each route_id and direction for each stop.
-  # Otherwise, return an empty list.
+  # For sections configured only to show predictions and not schedules, we may override
```

Comment:
Technically the behavior here does not depend on whether the section is configured to show schedules. Even if we are showing scheduled departures, "overnight departures" may still be generated and displayed (in particular for departures outside the current service day, or the "service ended" rows with the moon icon). In general though, appreciating the added detail in this comment!